### ISSUE TITLE
Getting Started tutorial TypeError: Cannot read property 'executeScript' of undefined

### ISSUE THREAD

**Issue by gordon-lim** [GoogleChrome/developer.chrome.com#119]
**Describe the bug**

Error handling response: TypeError: Cannot read property 'executeScript' of undefined

Highlighted line:       `chrome.scripting.executeScript({`

**To Reproduce**

```
let changeColor = document.getElementById('changeColor');
...
changeColor.onclick = function(element) {
  let color = element.target.value;
  chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
    chrome.scripting.executeScript({
      function: setTheColor
    });
  });
};
```

function setTheColor() {
  document.body.style.backgroundColor = "' + color + '";
}

**Expected behavior**
Change background color

**Comment by thsnyzkn** [comment]
Is there any development about this subject? :/ The whole Getting Started Tutorial is really hard to follow in general :(
